### DISCUSSION TITLE
Customize the message when no rows are found

### DISCUSSION THREAD

**Discussion by kevinrueda** [rappasoft/laravel-livewire-tables#1567] [Q&A] [answered]
The button is not rendered, I only see the literal string

```
public function configure(): void
    {
      ...
        $this->setEmptyMessage(new HtmlString('<a href="#" wire:click.prevent="..." class="btn btn-default shadow-md"><i class="far fa-laugh text-info"></i></a>'));
    }
```

**Comment by lrljoe** [comment]
This doesn't presently support HTML as far as I can remember, not a difficult fix tho, so can look at adding that into the next version.

**Reply by kevinrueda** [reply]
okay thank you very much for your help to the Livewire community

**Reply by lrljoe** [reply, marked as answer]
I've written the functionality, and docs, just need to write the tests now:

#1571
